.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+
+max-line-length = 80
+exclude =
```

Comment:
Shall we use whitelist or backlist? There are many folders that are not for python codes.

Reply:
It automatically ignore non .py files.